keyform/templates/keyform/base.html

```diff
@@ -24,10 +24,13 @@
                 <div class="title-bar">
                     <img src="{% static 'keyform/logo.jpg' %}"/>
                     <div class="title-bar-title">{% trans 'Keyform' %}</div>
+                    {% block links %}
```

Comment:
Improperly indented. Also, change it from "links" to "navigation." And remember to do that anywhere that block is being used.